lib/src/common/LinkingFlows.tsx

```diff
@@ -0,0 +1,68 @@
+export type LinkingFlow = {
+  name: string
+  icon?: string
+  displayName?: string
+  verification?: string
```

Comment:
```suggestion
  verification?: 'Verification' | 'Tweet'
```

This one named "verification" confuses me a bit a maybe ill get there down below as a keep reading